### DISCUSSION TITLE
CTX always returns undefined in Nextjs + Apollo Set Up

### DISCUSSION THREAD

**Discussion by themmyloluwaa** [vercel/vercel#4378] [General]
I'm having difficulties in integrating apollo and nextjs to work seamlessly together. I am configuring it to work for both query, mutation, and subscriptions but cookies set with js-cookie always returns undefined on page refresh, and context is always undefined.

**Comment by paulogdm** [comment]
Hello,

Please refer to Next.js discussions for further information: https://github.com/zeit/next.js/discussions.
If this is indeed a Vercel-only issue, provide more context so we can assist you further.
